opencsp/app/sofast/lib/MeasurementSofastFixed.py

```diff
@@ -33,20 +34,19 @@ def __init__(
             Optic to screen distance, meters
```

Comment:
Could you remove v_measure_point facet and dist_optic_facet from the doc string and replace with dist_optic_screen_measure?